server/items/itemModel.js

```diff
@@ -13,6 +13,16 @@ var ItemSchema = new mongoose.Schema ({
   itemLocation :{
     type: Object,
     required: true
+  },
+
+   itemLng: {
```

Comment:
Michael and I tried to remove documents without the extra itemLng and itemLat fields in our MongoDB schema. However, it is impossible for the backend server to select the itemLocation fields as an object. The itemLocation (typeof object) still has 'lng' and 'lat'(string) properties with number values. The back-end server doesn't recognize our itemLocation.lng and  itemLocation.lat with one line code. We tried JSON.parse and JSON.stringify too but didn't work. Adding the two additional properties simplified things and now we can remove the lat and lng.